### ISSUE TITLE
about adaptive_batchnorm

### ISSUE THREAD

**Issue by yuanhaorannnnnn** [ptrblck/pytorch_misc#13]
hello, I've read a lot discussions about pytorch from you, it was really helpful.
here I have a doubt about the implementation of adaptative batchnorm, why you use a * x + b * bn(x)  to do it? Could you explain a bit?

**Comment by ptrblck** [comment]
The implementation should be based on [Fast Image Processing with Fully-Convolutional Networks](https://arxiv.org/pdf/1709.00643.pdf) - see section 3.3 and [this discussion](https://discuss.pytorch.org/t/adaptive-normalization/9157).

Closing, but feel free to reopen in case you see any issues or post in the linked thread for more discussion.
